### DISCUSSION TITLE
storage: how to display image in html

### DISCUSSION THREAD

**Discussion by alanbar** [supabase/supabase#1820] [Questions] [answered]
Able to store images successfully in storage, but how do I display them in a svelte program?
Is there some sample code available?

**Comment by inian** [comment, marked as answer]
Does this repo help @alanbar? https://github.com/one-aalam/svelte-starter-kit/tree/auth-supabase
The easiest way to display an image in HTML is to [create a signed URL](https://supabase.io/docs/reference/javascript/storage-from-createsignedurl) and add it to your img tag. We are launching public buckets this week, so you can directly add objects in public buckets in your img tag without creating a signed URL.

**Reply by inian** [reply]
For anyone coming across this, we have [launched public buckets](https://supabase.io/docs/guides/storage#accessing-objects) now. You should be able to access assets in a public bucket without a token or a Authorization header.
